Violin.m

```diff
@@ -195,7 +201,15 @@
             
             %% Plot the data points within the violin area
             if length(density) > 1
-                jitterstrength = interp1(value, density*width, data);
+                try

```

Comment:
I have a feeling that there might exist a better solution than try/catch, by checking the preconditions for interp1 manually, or indeed sorting the values manually.

But maybe that solution would add too much complexity, which might not be warranted for this simple application.